---
content_type: page
description: This section provides the schedule of lecture topics and brief summaries
  of each lecture.
learning_resource_types:
- Lecture Notes
ocw_type: CourseSection
title: Lecture Summaries
uid: 01d35c19-d454-70ba-46b1-96856df0e56c
---

{{< tableopen >}}
{{< theadopen >}}
{{< tropen >}}
{{< thopen >}}
SES #
{{< thclose >}}
{{< thopen >}}
TOPICS
{{< thclose >}}
{{< thopen >}}
LECTURE SUMMARIES
{{< thclose >}}

{{< trclose >}}

{{< theadclose >}}
{{< tropen >}}
{{< tdopen >}}
1
{{< tdclose >}}
{{< tdopen >}}
Introduction to the class and topic
{{< tdclose >}}
{{< tdopen >}}
The instructors and students will introduce themselves. The instructors will discuss the administrative aspects of the course and give a lecture that presents a general overview of systems biology and, in particular, the aspects that will be the focus of the course. This will be the only lecture given in the course and we hope, nonetheless, for an interactive environment during the first day.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
2
{{< tdclose >}}
{{< tdopen >}}
Simple synthetic networks
{{< tdclose >}}
{{< tdopen >}}
Much of the recent interest in systems biology has been motivated by the need to understand how simple artificially constructed genetic networks operate in the cell. Early efforts included the production of simple networks capable of producing genetic oscillators and toggle switches. This week we will discuss two examples of artificial networks that led to many of the recent attempts at understanding network function and the consequences of gene expression variability.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
3
{{< tdclose >}}
{{< tdopen >}}
Noise in gene expression I
{{< tdclose >}}
{{< tdopen >}}
One of the first findings in systems biology was that gene expression is a fundamentally stochastic phenomenon. In particular, it appears that the randomness inherent to the biochemical processes involved in gene expression can lead to significant cell-to-cell variability in the numbers of proteins and mRNAs, leading to "non-genetic individuality" among genetically identical organisms. This week, we discuss a classic study of variability in bacterial gene expression and introduce the concepts of extrinsic and intrinsic noise. We also will read a paper that discusses some of the theoretical underpinnings of stochastic chemical kinetics and present concepts that have been widely applied to stochastic gene expression.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
4
{{< tdclose >}}
{{< tdopen >}}
Noise in gene expression II
{{< tdclose >}}
{{< tdopen >}}
Lately, a number of studies have indicated that stochastic gene expression in eukaryotic cells can result in even greater variability than that observed in bacteria. This variation is thought to happen because transcription in these organisms occurs in bursts rather than at a steady rate, leading to very broad population distributions of mRNAs and proteins. This week we will discuss two papers that examine stochastic gene expression in eukaryotes. One is a classic study of variability in cultured cells, and another is a study that utilizes quantitative PCR-based methods to count mRNAs in individual cells.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
5
{{< tdclose >}}
{{< tdopen >}}
Noise in gene expression III
{{< tdclose >}}
{{< tdopen >}}
Now that stochastic gene expression has been established as being an important biological effect, researchers are delving ever more deeply into the process of gene expression itself. This analysis includes trying to understand what biochemical reactions are the most prone to stochastic behavior and thus contribute the most to cellular variability. This week's papers include a theoretical examination of the sources of stochastic gene expression as well as a remarkable real-time visualization of stochastic gene expression.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
6
{{< tdclose >}}
{{< tdopen >}}
Structure of biological networks
{{< tdclose >}}
{{< tdopen >}}
One of the central goals of systems biology is to understand how the wiring of genetic networks allows them to perform cellular functions. While much is now known about what sets of genes interact, only recently have we begun to understand why those particular sets of interactions are useful. Through this week's papers, we will learn about efforts to find functional units ("network motifs") within large gene interaction networks and also check whether a particularly common motif actually work as theory predicts.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
7
{{< tdclose >}}
{{< tdopen >}}
Network evolution and adaptation
{{< tdclose >}}
{{< tdopen >}}
We often discuss genetic networks from a functional standpoint, but we must always remember that these networks exist in the context of evolutionary challenges and constraints. In this week's papers we will see two very different responses to novel environmental challenges.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
8
{{< tdclose >}}
{{< tdopen >}}
Chemotaxis I
{{< tdclose >}}
{{< tdopen >}}
Microorganisms live in an environment very different from the world that we are used to. This week we will discuss a paper describing the basic physical challenges facing a cell. Although physics in the regime of small size and high viscosity make some things impossible, microbes have nevertheless found some surprising solutions to the challenges they face.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
9
{{< tdclose >}}
{{< tdopen >}}
Chemotaxis II
{{< tdclose >}}
{{< tdopen >}}
This week we discuss two papers that probe the ability of E. coli to swim towards food sources. For several decades it has been known that this chemotaxis occurs by alternating "runs," in which the cell swims straight, followed by "tumbles," in which the orientation of the cell is randomized. The cell is able to move towards food sources by altering the frequency of these tumbles depending upon whether things are improving or getting worse. This week's papers show that because the steady-state frequency of tumbling is independent of the concentration of food sources (robustness), the cell is able to respond to gradients over a wide range of concentrations.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
10
{{< tdclose >}}
{{< tdopen >}}
Circadian oscillations
{{< tdclose >}}
{{< tdopen >}}
Circadian oscillations are 24-hour cellular oscillations that can be entrained by the day-night cycle—they are responsible for many phenomena, including, for instance, jet lag. This week, we discuss a paper that examines the circadian clock in cyanobacteria and another paper about the very different circadian clock used by mammals.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
11
{{< tdclose >}}
{{< tdopen >}}
Field trip
{{< tdclose >}}
{{< tdopen >}}
This week, we will take a field trip to the laboratory of {{% resource_link "ae5453e1-aae7-42e6-8967-64b63fa2798b" "Sunney Xie in the Chemistry and Chemical Biology Department at Harvard University" %}} (see paper from week 5 in {{% resource_link d90bf707-ed30-9766-70f1-b4090aeef696 "Readings" %}}). The goal is to expose students to current work in the field while also highlighting some of experimental methods that systems biologists use in their research.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
12
{{< tdclose >}}
{{< tdopen >}}
Noise in development
{{< tdclose >}}
{{< tdopen >}}
The developmental of an organism from a single cell to an adult is one of the most complex biological programs in existence. Yet, despite its complexity, the program usually executes with remarkable fidelity, a prime example of biological robustness. However, there are a few important instances where variability, arising from stochastic gene expression or otherwise, can lead the program to alternate endings. This week, we explore one paper in which stochastic gene expression is used by an organism to produce useful variability in development and another paper in which a different mechanism leads to randomly determined cell fates in _C. elegans_.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
13
{{< tdclose >}}
{{< tdopen >}}
Synthetic biology
{{< tdclose >}}
{{< tdopen >}}
At the beginning of this course we read several papers in which very simple genetic networks were constructed. These projects allowed experimentalists to test models of gene expression and also provided a step towards engineering entirely new cellular functions. In our last week we will discuss two short papers describing the increasingly ambitious efforts of synthetic biologists to bend nature to man's will.
{{< tdclose >}}

{{< trclose >}}

{{< tableclose >}}